src/main/antora/modules/websvcs/pages/xml-websvcs/jakarta-xml-binding.adoc

```diff
@@ -0,0 +1,6 @@
+== Jakarta XML Binding
+
+Jakarta XML Binding (JAXB) provides a way to convert Java objects to XML and vice versa. It is a part of the Jakarta EE platform.
+
+* https://jakarta.ee/specifications/xml-binding/[Jakarta XML Binding Specification]
+* https://jakarta.ee/specifications/xml-binding/3.0/[Example using Jakarta REST with Jakarta XML Binding]
```

Comment:
The last link isn't an example. Here's the example I had in mind: https://jakartaee.github.io/jakartaee-documentation/jakartaee-tutorial/current/websvcs/rest-advanced/rest-advanced.html#_using_jakarta_rest_with_jakarta_xml_binding. (You need to provide an xref link within the tutorial, though  -- not an absolute URL).

Reply:
Also provide a link to this example in the older version of the tutorial (again, use an xref link not this actual URL): https://jakartaee.github.io/jakartaee-documentation/jakartaee-tutorial/9.1/websvcs/jaxws/jaxws.html